frontend/src/components/loginOutButton/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { useAuth0 } from '../../react-auth0-spa';
+
+// Button to log in user via Auth0
+// The button is disabled when Auth0 authentication is loading
+
+const LoginOutButton = () => {
+  const {
+    loading,
+    isAuthenticated,
+    loginWithRedirect,
+    logout,
+    user,
+  } = useAuth0();
+
+  return (
+    <>
```

Comment:
I'd maybe make this a div, with a descriptive className, so that it's easier to identify in the future